lib/build.py

```diff
@@ -356,11 +356,17 @@ def __init__(self, *args):
          paths = [variables_sub(i, env.env_build)
                   for i in ch.tree_child_terminals(self.tree, "copy_shell",
                                                    "WORD")]
-         self.srcs = paths[:-1]
-         self.dst = paths[-1]
+      elif (ch.tree_child(self.tree, "copy_list") is not None):
+         paths = [variables_sub(i, env.env_build)
+                  for i in ch.tree_child_terminals(self.tree, "copy_list",
+                                                   "STRING_QUOTED")]
+         for i in range(len(paths)):
+            paths[i] = paths[i][1:-1] #strip quotes
       else:
-         assert (ch.tree_child(self.tree, "copy_list") is not None)
-         self.unsupported_yet_fatal("list form", 784)
+         ch.FATAL("not valid form of COPY")
```

Comment:
Getting here is a logic error, so IMO crashing (with a traceback) is probably better than a user-facing fatal error.

```suggestion
         assert (False, "unreachable code reached")
```